core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -119,6 +119,9 @@ THE SOFTWARE.
                       <j:if test="${p.isForNewerJava()}">
                         <div class="alert alert-danger">${%javaWarning(p.minimumJavaVersion)}</div>
                       </j:if>
+                      <j:if test="${p.fixesSecurityVulnerabilities()}">
+                        <div class="alert alert-warning">${%Applying this update will address security vulnerabilities currently affecting Jenkins.}</div>
```

Comment:
nit: maybe this would be better
```suggestion
                        <div class="alert alert-warning">${%Applying this update will address current security vulnerabilities.}</div>
```

Reply:
Agreed. "affecting" may imply that we know that the instance is affected, but in fact we don't. Some vulns may be a problem only in particular configurations

Reply:
```suggestion
                        <div class="alert alert-warning">${%Applying this update will address security vulnerabilities in the currently installed version.}</div>
```